simpleflow/execute.py

```diff
@@ -409,7 +411,25 @@ def main():
         metavar='N',
         help='error file descriptor',
     )
+    parser.add_argument(
+        '--kill-children',
+        action='store_const',
+        const=True,
+        default=False,
+        help='kill child processes on exit',
+    )
     cmd_arguments = parser.parse_args()
+
+    def kill_child_processes():
+        process = psutil.Process(os.getpid())
+        children = process.children(recursive=True)
+
+        for child in children:
```

Comment:
(note: the test catches the exception from exactly this call: https://github.com/botify-labs/simpleflow/blob/59c8bc8b17b0705c5b78fc92775be4d313e401b5/tests/test_simpleflow/test_execute.py#L311 )